lib/zendesk_apps_support/package.rb

```diff
@@ -205,18 +296,40 @@ def non_tmp_files
       files
     end
 
+    def commonjs_modules
+      return {} unless has_lib_js?
+
+      lib_files.each_with_object({}) do |file, modules|
+        name          = file.relative_path.gsub!(/^lib\//, '')
+        content       = file.read
+        modules[name] = content
+      end
+    end
+
     def file_exists?(path)
       File.exist?(file_path(path))
     end
 
+    def deep_hash_merge(h, another_h)
+      result_h = h.dup
+      another_h.each do |key, value|
+        if h.has_key?(key) && h[key].is_a?(Hash) && value.is_a?(Hash)
+          result_h[key] = deep_hash_merge(h[key], value)
+        else
+          result_h[key] = value
+        end
+      end
+      result_h
+    end
+
     def read_file(path)
       File.read(file_path(path))
     end
 
-    def read_json(path, symbolize_names = true)
+    def read_json(path)
```

Comment:
it was symbols in ZAT and strings in ZAM. I choose strings. Also Tobi told me it could be a security hole?